src/store/channels/saga.ts

```diff
@@ -102,14 +110,9 @@ export function* clearChannels() {
 }
 
 export function* saga() {
-  yield takeEveryFromBus(yield call(conversationsChannel), ChannelEvents.MessagesLoadedForChannel, (event) =>
-    markChannelAsReadIfActive(event.channelId)
-  );
-  yield takeEveryFromBus(yield call(conversationsChannel), ChannelEvents.MessagesLoadedForConversation, (event) =>
-    markConversationAsReadIfActive(event.channelId)
-  );
-
   yield takeLatest(SagaActionTypes.JoinChannel, joinChannel);
+  yield takeLatest(SagaActionTypes.OpenChannel, openChannel);
+  yield takeLatest(SagaActionTypes.OpenConversation, openConversation);
```

Comment:
Would be good to use the new format we discussed where these parse out the `payload` so that the sagas can just take the information they need.

This sort of thing: `takeLatest(SagaActionTypes.OpenConversation, ({payload}) => yield openConversation(payload.channelId));`

That way the other sagas don't have to format the arguments in weird ways when they call `openConversation`

Reply:
could you please check if this makes sense 😅 : https://github.com/zer0-os/zOS/pull/991/commits/25d3ad2a6856e4604970153a6ed8f05aead344ba

Reply:
Yep, that makes sense. Although, now that I see it again, I wonder if that function has to `yield` in order for the `takeLatest` to properly work. :thinking:

Reply:
In this case it's fine because I wouldn't expect anything to happen rapidly enough for multiple calls to step over each other but we should definitely verify this. We might have to tweak the pattern.